libpod/events/events_linux.go

```
@@ -0,0 +1,15 @@
+package events
+
+import "strings"
+
+// NewEventer creates an eventer based on the eventer type
+func NewEventer(event *Event, options EventerOptions) Eventer {
+	var eventer Eventer
+	switch strings.ToUpper(options.EventerType) {
+	case strings.ToUpper(LogFile.String()):
+		eventer = EventJournalD{event, options}
+	default:
```

Don't make this default. Default should return an error.